pkg/tests/apis/scopes/scopes_test.go

```diff
@@ -0,0 +1,137 @@
+		AppModeProduction: false, // required for experimental APIs
+		EnableFeatureToggles: []string{
+			featuremgmt.FlagGrafanaAPIServerWithExperimentalAPIs, // Required to start the example service
+		},
+	})
+
+	t.Run("Check discovery client", func(t *testing.T) {
+		disco := helper.NewDiscoveryClient()
+		resources, err := disco.ServerResourcesForGroupVersion("scope.grafana.app/v0alpha1")
+		require.NoError(t, err)
+
+		v1Disco, err := json.MarshalIndent(resources, "", "  ")
+		require.NoError(t, err)
+		fmt.Printf("%s", string(v1Disco))
+
+		require.JSONEq(t, `{
+			"kind": "APIResourceList",
+			"apiVersion": "v1",
+			"groupVersion": "scope.grafana.app/v0alpha1",
+			"resources": [
+			  {
+				"name": "scopedashboardbindings",
+				"singularName": "scopedashboardbinding",
+				"namespaced": true,
+				"kind": "ScopeDashboardBinding",
+				"verbs": [
+				  "create",
+				  "delete",
+				  "deletecollection",
+				  "get",
+				  "list",
+				  "patch",
+				  "update",
+				  "watch"
+				]
+			  },
+			  {
+				"name": "scopes",
+				"singularName": "scope",
+				"namespaced": true,
+				"kind": "Scope",
+				"verbs": [
+				  "create",
+				  "delete",
+				  "deletecollection",
+				  "get",
+				  "list",
+				  "patch",
+				  "update",
+				  "watch"
+				]
+			  }
+			]
+		  }`, string(v1Disco))
+	})
+
+	t.Run("Check create and list", func(t *testing.T) {
+		// Scope create+get
+		scopeClient := helper.GetResourceClient(apis.ResourceClientArgs{
+			User:      helper.Org1.Admin,
+			Namespace: "default", // actually org1
+			GVR: schema.GroupVersionResource{
+				Group: "scope.grafana.app", Version: "v0alpha1", Resource: "scopes",
+			},
+		})
+		s0, err := scopeClient.Resource.Create(ctx,
+			helper.LoadYAMLOrJSONFile("testdata/example-scope.yaml"),
+			metav1.CreateOptions{},
+		)
+		require.NoError(t, err)
+		require.Equal(t, "example", s0.GetName())
+		s1, err := scopeClient.Resource.Get(ctx, "example", metav1.GetOptions{})
+		require.NoError(t, err)
+		require.Equal(t,
+			mustNestedString(s0.Object, "spec", "title"),
+			mustNestedString(s1.Object, "spec", "title"),
+		)
+
+		// Create bindings
+		scopeDashboardBindingClient := helper.GetResourceClient(apis.ResourceClientArgs{
+			User:      helper.Org1.Admin,
+			Namespace: "default", // actually org1
+			GVR: schema.GroupVersionResource{
+				Group: "scope.grafana.app", Version: "v0alpha1", Resource: "scopedashboardbindings",
+			},
+		})
+		_, err = scopeDashboardBindingClient.Resource.Create(ctx,
+			helper.LoadYAMLOrJSONFile("testdata/example-scope-dashboard-binding-abc.yaml"),
+			metav1.CreateOptions{},
+		)
+		require.NoError(t, err)
+		_, err = scopeDashboardBindingClient.Resource.Create(ctx,
+			helper.LoadYAMLOrJSONFile("testdata/example-scope-dashboard-binding-xyz.yaml"),
+			metav1.CreateOptions{},
+		)
+		require.NoError(t, err)
+
+		found, err := scopeDashboardBindingClient.Resource.List(ctx, metav1.ListOptions{})
```

Comment:
@kylebrandt -- would be good to exercise ListOptions with the fieldSelectors we are implementing